docs/usage/ruby.md

```diff
@@ -17,10 +17,12 @@ Renovate supports upgrading dependencies in Bundler's Gemfiles and their accompa
 
 ## Warnings
 
-Renovate doesn't update dependencies without a version constraint.
+When using `"rangeStrategy": "bump"`, Renovate doesn't update dependencies without a version constraint.
```

Comment:
Also rangeStrategy=replace? Essentially anything except update-lockfile

Reply:
I adjusted the README with that suggestion, and thought that it would be easier to read if the two blocks are switched (first the specific case, then the else-case).